h2o-r/h2o-DESCRIPTION.template

```diff
@@ -57,7 +57,12 @@ Suggests: ggplot2,
           data.table (>= 1.9.8),
           rgl (>= 0.100.19),
           plot3Drgl (>= 1.0.1),
-          survival
+          survival,
+          DT,
```

Comment:
`DT`, `htmltools`, `plotly` are required for html output of explanation.
`IRdisplay`, `repr` is required for html rendering in Jupyter notebook with R kernel. (Those packages are dependencies of R jupyter kernel).

Without adding them to `Suggests`, R check as CRAN would complain.

Reply:
It's ok to add them to Suggests.